### PR TITLE
Add a post calling for blog posts to feed into the 2019 Roadmap RFC

### PR DESCRIPTION
**Pull request by kategengler**
Call for 2019 Roadmap posts^

### CODE REVIEW

File: source/2019-05-20-ember-2019-roadmap-call-for-posts.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Ember's 2019 Roadmap: Call for Blog Posts"
+author: Katie Gengler
+tags: Recent Posts, Roadmap, 2019
+responsive: true
+---
+
+Ember in 2018 (and 2019) has been more exciting than ever! We've been working diligently to advance the project based on our [2018 Roadmap](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md), but while we work on that, it is time to think about the future.
```
Comment (MelSumner) [inline source/2019-05-20-ember-2019-roadmap-call-for-posts.md:8]: Ember in 2018 (and 2019) has been more exciting than ever! We've been working diligently to advance the project based on our [2018 Roadmap](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md), but while we work to finish that, it is time to think about the future.

File: source/2019-05-20-ember-2019-roadmap-call-for-posts.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+The 2018 Roadmap led to the ["Octane Edition" of Ember, currently in preview](https://emberjs.com/editions/octane/). Octane [includes many features](https://github.com/emberjs/ember.js/issues/17234), most of which have already landed in a stable release. As the edition nears completion, we want the community to think beyond Octane to the next steps for Ember. The resulting Roadmap for 2019 will not necessarily include an edition but will outline what we will be working toward. 
+
+**The Roadmap process is part of our on-going effort to have shared, clear, and published project-wide goals.**
+
+# #EmberJS2019
+
+To contribute a post: Tweet a link to the post with the hashtag #EmberJS2019 or email a link to roadmap@emberjs.com. These posts will be collected and categorized, and each one will be read by those working to draft the Roadmap RFC.
+
+### _Format_
```
Comment (MelSumner) [inline source/2019-05-20-ember-2019-roadmap-call-for-posts.md:26]: this should be an H2 if the one before it is an H1

File: source/2019-05-20-ember-2019-roadmap-call-for-posts.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+Your post can be on any online writing platform, but we suggest:
+
+* A post on your personal or company blog
+* A [GitHub gist](https://gist.github.com/)
+
+### _Topic ideas_
```
Comment (MelSumner) [inline source/2019-05-20-ember-2019-roadmap-call-for-posts.md:32]: Same as above

**Review by MelSumner — CHANGES_REQUESTED** [review]

File: source/2019-05-20-ember-2019-roadmap-call-for-posts.md (hunk 2 above)
Comment (kategengler) [reply source/2019-05-20-ember-2019-roadmap-call-for-posts.md:26]: /shrug I used same as last year

Comment (MelSumner) [reply source/2019-05-20-ember-2019-roadmap-call-for-posts.md:26]: I'm sorry I didn't catch it then- it's not good for accessibility.

File: source/2019-05-20-ember-2019-roadmap-call-for-posts.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+# What's a Roadmap RFC?
+
+The Roadmap will outline the community's priorities for the next year. It will be presented as an RFC and follow [the normal RFC process](https://github.com/emberjs/rfcs). The RFC will be written based on community input--the blog posts we've requested and other community discussion--and curation from the core teams.
+
+This is will be our second annual Roadmap RFC. To see the results of the process, see [the 2018 Roadmap RFC](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md). 
+
+The 2018 Roadmap led to the ["Octane Edition" of Ember, currently in preview](https://emberjs.com/editions/octane/). Octane [includes many features](https://github.com/emberjs/ember.js/issues/17234), most of which have already landed in a stable release. As the edition nears completion, we want the community to think beyond Octane to the next steps for Ember. The resulting Roadmap for 2019 will not necessarily include an edition but will outline what we will be working toward.
```
Comment (jenweber) [inline source/2019-05-20-ember-2019-roadmap-call-for-posts.md:18]:
```suggestion
The 2018 Roadmap led to the ["Octane Edition" of Ember, currently in a public preview period](https://emberjs.com/editions/octane/). Octane [includes many features](https://github.com/emberjs/ember.js/issues/17234), most of which have already landed in stable releases. As the edition nears completion, we ask the community to think beyond Octane to the next steps for Ember. The resulting Roadmap for 2019 will not necessarily include an edition but will outline what we will be working toward. 
```

File: source/2019-05-20-ember-2019-roadmap-call-for-posts.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+**Late June**: Post the Roadmap RFC on GitHub
+
+**Mid July**: assuming we've reached consensus, and following the normal RFC process, accept final Roadmap
+
+This is our second call for blog posts. For inspiration, or to better understand the process, you may want to look at the 2018 blog posts, [as collected by community member Lennex Zinyando](https://github.com/zinyando/emberjs2018-posts) or directly on [twitter](https://twitter.com/search?q=%23emberjs2018), and the [2018 Roadmap RFC](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md).
+
+# Thanks
+
+Many thanks also to the Rust Core team and the Rust community, from whom we are still learning and adapting process.
```
Comment (jenweber) [inline source/2019-05-20-ember-2019-roadmap-call-for-posts.md:70]:
```suggestion
Thank you to everyone who participates this year, whether by writing, reading, or sharing! Many thanks also to the Rust Core team and the Rust community, from whom we are still learning and adapting processes.
```

File: source/2019-05-20-ember-2019-roadmap-call-for-posts.md (hunk 1 above)
Comment (jenweber) [reply source/2019-05-20-ember-2019-roadmap-call-for-posts.md:8]:
```suggestion
It's time to think about the future of Ember, and we need your ideas.

In all of Ember's history, 2018 and 2019 were more exciting than ever, and we want to know where you think that momentum could take us. The core teams and community have been working diligently to advance the project based on our [2018 Roadmap](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md), but what should the next step be?
```

**Review by jenweber — APPROVED** [review]
Some optional edits, just trying to jazz it up some.

**Pull request merged**